bots/activity.bot.js

```diff
@@ -76,27 +87,28 @@ const checkChannelCooldown = (channelKey)=>{
  * thresholds and notify #general about them.
  */
 const checkForActivityBursts = ()=>{
-	_.forEach(messageTimestamps, (timestamps, channelKey)=>{
+	messageTimestamps = _.reduce(messageTimestamps, (res, timestamps, channelKey)=>{
```

Comment:
Pro-tip: for cases like this where you're "transforming" an object into a new object (or an array into a new array), instead of "reducing" an object or array down into a _single_ value, you can tighten things up a bit by using `_.transform` instead.

It's basically the same as reduce, but you don't have to specify an initial value for the accumulator if it's the same type as the original, and you don't have to keep returning the same accumulator over and over again.

Reply:
omg 😍😍😍😍😍

Reply:
https://github.com/stolksdorf/pico-utils/blob/master/iterators.js#L5

here it is as a one-liner ;)